velox/expression/ComplexWriterTypes.h

```diff
@@ -428,7 +428,7 @@ class MapWriter {
 
   std::tuple<PrimitiveWriter<K, false>, PrimitiveWriter<V>> operator[](
       vector_size_t index) {
-    static_assert(std_interface, "operator [] not allowed for this map");
+    // static_assert(std_interface, "operator [] not allowed for this map");
```

Comment:
It would be nice to move current content of PR decription here, and update the PR description to be more precise.

Reply:
I'm not sure why this line is here. 
If it's unnecessary, I can delete it. Therefore, we don't need to include any description in this section.

Reply:
yes, This pull request is dependent on resolving this issue.